### PR TITLE
Added Work Allocation test for  Working scenario

### PR DESCRIPTION
**Pull request by phildebrah**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by will-craig — APPROVED** [review]

**Review by will-craig — APPROVED** [review]

File: UI/Selenium/Pages/ManageWorkAllocationPage.cs [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+        public static By EditAvailability = By.Id("edit-availability"); 
+        
+         public static By ManageWorkAllocation = By.Id("manageWorkAllocationBtn");
+    
+        // Upload working hours availability
+        public static By UploadWorkingHoursOrNonAvailability = By.Id("upload-availability");
+        public static By UploadCSVFile = By.XPath("//input[@id='working-hours-file-upload']");
+        public static By UploadAvailabilityHoursButton = By.CssSelector("#working-hours-file-upload-error .govuk-button");
+        public static By TeamWorkingHoursUploadedSuccessfully = By.CssSelector("div#file-upload-result > p");
+        //public static By TeamNonAvailabilityHoursUploadedSuccessfully = By.XPath("//p[normalize-space()='Team non-availability hours uploaded successfully'");
```
Comment (shaed-parkar) [inline UI/Selenium/Pages/ManageWorkAllocationPage.cs:25]: Please remove the commented code

File: UI/Selenium/Pages/ManageWorkAllocationPage.cs [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+        }
+        
+        public static void UploadNonWorkHoursFile(IWebDriver webdriver, string filePath)
+        {
+            var fullPath = System.IO.Path.GetFullPath(filePath);
+            System.IO.File.Exists(filePath).Should().BeTrue();
+            webdriver.FindElement(UploadNonAvailabilityHours).SendKeys(fullPath);
+            webdriver.FindElement(UploadNonAvailabilityHoursButton).Click();
+            
+            //ExtensionMethods.WaitForElementVisible(webdriver, TeamNonAvailabilityHoursUploadedSuccessfully);
```
Comment (shaed-parkar) [inline UI/Selenium/Pages/ManageWorkAllocationPage.cs]: and removed the commented code here too please

File: UI/Selenium/Pages/ManageWorkAllocationPage.cs [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+        public static By AllocateTo = By.CssSelector("#user-menu #users");
+        public static By OnlyAllocatedHearing = By.Id("is-unallocated");
+        public static By Search = By.XPath("//main[@id='main-content']/app-work-allocation/div/app-allocate-hearings/details/div/div[4]/button");
+        
+       /// <summary>
+       /// 
+       /// </summary>
+       /// <param name="webdriver"></param>
+       /// <param name="filePath"></param>
+
```
Comment (shaed-parkar) [inline UI/Selenium/Pages/ManageWorkAllocationPage.cs:73]: could you remove the empty line here please

File: UI/Selenium/Features/Web Features/ManageWorkAllocation.feature [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+Scenario: Upload Non Availability hours
+	Given I log in as "auto_aw.videohearingsofficer_02@hearings.reform.hmcts.net"	
+	Then the page should be accessible
+	When i click on Manage Work Allocation Dashboard
+	Given i click on Upload Workhours and non Availability
+	When i click on Upload non Availability hours
+	Then file is uploaded successfully non availability hours
+				
+
```
Comment (shaed-parkar) [inline UI/Selenium/Features/Web Features/ManageWorkAllocation.feature:27]: can probably remove the empty lines here too

File: UI/Selenium/Pages/ManageWorkAllocationPage.cs (hunk 1 above)
Comment (phildebrah) [reply UI/Selenium/Pages/ManageWorkAllocationPage.cs]: Unwanted elements remove

**Review by shaed-parkar — APPROVED** [review]

File: azure-pipelines.sds.release.yml [hunk 5]
```diff
@@ -82,30 +82,8 @@ stages:
                 inputs:
                   command: test
                   projects: "$(projectPath)/*.csproj"
-                  arguments: "--configuration Release --filter TestCategory=Smoketest"
-                  publishTestResults: true
-
-              # - task: DotNetCoreCLI@2
```
Comment (shaed-parkar) [inline azure-pipelines.sds.release.yml:88]: please delete all this commented tasks if we don't need it

File: UI/Selenium/TestData/Good.csv [hunk 6]
```diff
@@ -0,0 +1,4 @@
+Username,Monday,,Tuesday,,Wednesday,,Thursday,,Friday,,Saturday,,Sunday,
+,Start,End,Start,End,Start,End,Start,End,Start,End,Start,End,Start,End
+oliver.scott@hearings.reform.hmcts.net,08:00,16:00,10:00,17:30,08:00,17:30,08:00,17:01,09:00,17:00,,,,
```
Comment (shaed-parkar) [inline UI/Selenium/TestData/Good.csv:3]: Please replace these emails with made up ones

**Review by shaed-parkar — APPROVED** [review]
Maybe double check about usernames in the repo

**Comment by phildebrah** [comment]
Work allocation CSV upload successfully

**Pull request merged**